src/styles/styles.js

```diff
@@ -698,6 +698,11 @@ const styles = {
         borderRadius: variables.componentBorderRadiusNormal,
     },
 
+    textInputMultiline: {
+        scrollPadding: '23px 0 0 0',
+        overflowY: 'auto',
```

Comment:
Is this necessary?

Reply:
Actually it’s not, there’s a warning from FF because not detect a scroll of the input and probably the offset scroll not detected.

It’s just some attempt to make it work on your end before.

It’s safe to remove the overflow.

Reply:
Please remove it